archethic-foundation/archethic-node
# Update chain explorer list display 452

# Description

- Add sort by desc date order
- Add transaction date in transactions chain

<img width="1713" alt="Screen Shot 2022-09-20 at 11 42 05" src="https://user-images.githubusercontent.com/34317793/191224976-5f775110-670b-467c-b690-cc2c84b806b2.png">

Fixes # ([452](https://github.com/archethic-foundation/archethic-node/issues/452))

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Go to the explorer
- Click on " Chains " in the navigation bar
- Insert a transaction address and search for it
- Check that the transaction date is available for each of them
- Check that all transactions are sorted by desc date order.

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules
